api/dp-openapi.yaml

```diff
@@ -1020,6 +1189,69 @@ components:
       items:
         type: string
 
+    proof_page_text:
+      type: object
+      properties:
+        text:
+          description: the page text from current round.
+          type: string
+
+    proof_page_data:
+      type: object
+      properties:
+        text:
+          description: the page text. If page state is 'out' from OCR or previous round, else from current round
+          type: string
+        pagestate:
+          description: the page state, for checking continuity. Either out or temp
+          type: string
+        saved:
+          description: true if the page state is 'temp'
+          type: boolean
+        pagename:
+          description: the filename of the page with extension
+          type: string
+        image_url:
+          description: url of the image of the page
+          type: string
+        pageNum:
+          description: page "number" without extension
+          type: string
+        roundInfoArray:
```

Comment:
Ditto no snakeCase - consider `round_info_array`.

Reply:
done this.